src/modules/EventModule.cpp

```diff
@@ -429,7 +429,7 @@ EventModule::EventModule(UmikoBot* client) : Module("event", true)
 		CurrencyModule* currencyModule = static_cast<CurrencyModule*>(UmikoBot::Instance().GetModuleByName("currency"));
 		auto& authorCurrency = currencyModule->guildList[channel.guildId()][currencyModule->getUserIndex(channel.guildId(), authorID)];
 
-		if (authorCurrency.currency() <= 0)
+		if (authorCurrency.currency() <= Currency(0))
```

Comment:
Do you really have to explicitly cast to Currency?

Reply:
You have to explicitly construct a Currency object. It doesn't implicitly do it. If I add operators that do, it will not compile on MSVC. See C2666